azure-pipelines.yml

```diff
@@ -82,23 +100,23 @@ jobs:
       ./bin/nrnivmodl-core $(Build.Repository.LocalPath)/test/integration/mod
     env:
       CMAKE_PKG: 'cmake-3.10.2-Linux-x86_64'
-    displayName: 'Build and Run Integration Tests with ISPC compiler'
-  - script: |
-      export PATH=$(pwd)/$CMAKE_PKG/bin:/home/vsts/.local/bin:$PATH
-      export CXX='g++-8'
-      sudo apt-get install -y python3.7-venv
-      python3.7 -m pip install -U scikit-build
-      python3.7 setup.py sdist bdist_wheel -j 6
-      rm -rf venv _skbuild
-      python3.7 -m venv venv
-      source ./venv/bin/activate
-      pip install --upgrade pip
-      pip install dist/*.whl
-      nmodl $(Build.Repository.LocalPath)/test/integration/mod/cabpump.mod sympy --analytic
-      rm cabpump.cpp
-    env:
-      CMAKE_PKG: 'cmake-3.10.2-Linux-x86_64'
-    displayName: 'Build and test pywheels'
+    displayName: 'Build CoreNEURON and Run Integration Tests with ISPC compiler'
+#  - script: |
+#      export PATH=$(pwd)/$CMAKE_PKG/bin:/home/vsts/.local/bin:$PATH
+#      export CXX='g++-8'
+#      sudo apt-get install -y python3.7-venv
+#      python3.7 -m pip install -U scikit-build
+#      python3.7 setup.py sdist bdist_wheel -j 6
+#      rm -rf venv _skbuild
+#      python3.7 -m venv venv
+#      source ./venv/bin/activate
+#      pip install --upgrade pip
+#      pip install dist/*.whl
+#      nmodl $(Build.Repository.LocalPath)/test/integration/mod/cabpump.mod sympy --analytic
+#      rm cabpump.cpp
```

Comment:
Just a note : when we will test wheel, we should make sure to run `cabpump.mod` and other mod files. These are added to see if generated binary/wheel has everything needed including sympy.

Reply:
Good point, i missed to add that into the `test_wheel.bash` script